packages/binding-file/test/.gitignore

```diff
@@ -0,0 +1 @@
+test.*
```

Comment:
You need that since you create test files, right? Am I wrong?
Does it make sense to add a comment?

Reply:
+ 1 for a comment. I was surprised to find here the ignore too.

Reply:
Sure, will add that :) In theory, we could also remove the `.gitignore` since the test files are cleaned up after the tests are finished, but I thought it might be good to avoid committing any test artifacts here.

Reply:
yes let's keep it to be 100% :)

Reply:
Would this work for you?

```suggestion
# Ignore auxiliary files created by the FileClient implementation tests
test.*
```

Reply:
Looks good. Maybe the comment can be even next to the test.* entry.. just in case other ignores are added below for some other reasons but we might not need to bother now 🙈

Reply:
I am fine either way also. Since it is your PR I let you decide ;-)